src/krishna/Assignment_13_indexOf.java

```diff
@@ -0,0 +1,68 @@
+package krishna;
+
+import java.util.Scanner;
+
+public class Assignment_13_indexOf {
+	
+	void findDupliChar(String provide) {
+		char ch = ' ';
+		System.out.println("The duplicate characters are : ");
+		boolean flag = true;
+		for (int index1 = 0; index1 < provide.length(); index1++) {
+			ch = provide.charAt(index1);
+			if (provide.indexOf(ch) == provide.lastIndexOf(ch)) {
+				flag = false;
+			}
+			if (flag == true)
+				System.out.println(ch);
+		}
+	}
+		
+	void uniqueChar(String test){
+		String temp = "";
+	    int i;
+	    for ( i = 0; i < test.length(); i++){
```

Comment:
this logic fails for "harsh" as first 'h' has index 0 when second 'h' go for indexOf then it will also return 0 right? indexOf method always returns the first occurrence char index.